src/index.css

```
@@ -1,18 +1,112 @@
+@import "@patternfly/patternfly/patternfly-addons.css";
+
+/* PF6 removed component-level theme="dark" prop; override tokens directly.
+   PF6 Page uses CSS grid with subgrid on the masthead, which forces stacking.
+   Override to flex to match PF5 inline masthead layout. */
+/* Style inline filter input inside select dropdowns */
+.pf-v6-c-menu .pf-v6-c-menu__content > .pf-v6-c-text-input-group {
+  padding: var(--pf-t--global--spacer--sm);
+  border-bottom: 1px solid var(--pf-t--global--border--color--default);
+  --pf-v6-c-text-input-group--BorderWidth: 0;
+  --pf-v6-c-text-input-group--BackgroundColor: transparent;
+  background-color: transparent;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)